app/views/accounts/_emails.html.erb

```diff
@@ -31,9 +31,10 @@
   <% end %>
 </div>
 <% if EmailPolicy.new(current_user).can_add_email? %>
-  <%= link_to(
-        prefix_with_plus(t('account.index.email_add')),
-        add_email_path,
-        class: 'usa-button usa-button--outline margin-top-2',
-      ) %>
+  <%= render ButtonComponent.new(
+        action: ->(content, type:, **tag_options) { link_to(content, add_email_path, **tag_options) },
```

Comment:
Couple usability thoughts:

- Excluding `type:` is not very obvious
   - Maybe we shouldn't be adding this in the ButtonComponent implementation, and instead rely on the consumer to pass it in? After all, this is how one would need to use a `<button>` element.
   - Maybe we could have a short-hand for this use case, like a `href:` keyword argument to `ButtonComponent`.
- It's a bit odd that the `action` implementation deals with how to use `content` as an argument, when we control what that content is going to be assigned to just a few lines below this.
   - One could argue we just specify that translation string here and drop the parameter.
   - Then again, "content" is a formal concept of ViewComponent, and we'd need to support the block form regardless for the default behavior of `action`.

Reply:
We can pass `&block` to the `action` and expect that it passes it along too right?

Reply:
Another idea to make excluding `type:` more obvious?
```suggestion
        action: ->(content, **tag_options) { link_to(content, add_email_path, **tag_options.except(:type)) },
```

Reply:
>We can pass `&block` to the `action` and expect that it passes it along too right?

To what end would you have in mind? I guess it could feel a bit more natural to do something like...

```rb
        action: ->(**tag_options, &block) do
          content_tag(:'lg-custom-button', **tag_options, data: { extra: '' }, &block)
        end,
```

>Another idea to make excluding type: more obvious?

It seems like an easy thing to miss that a developer must explicitly exclude it.

Another idea could be to add the default `type` only in the _default action implementation_:

```rb
action: ->(content, **tag_options) { button_tag(content, **tag_options, type: tag_type) },
```